charts/atlas-operator/Chart.yaml

```
@@ -19,6 +19,6 @@ maintainers:
     email: support@mongodb.com
 dependencies:
   - name: mongodb-atlas-operator-crds
-    version: "1.9.1"
+    version: "1.9.2"
```

@helderjs I guess the CRDs are fine right?

```suggestion
    version: "1.9.1"
```